### ISSUE TITLE
Crashes under Wayland in Linux

### ISSUE THREAD

**Issue by Athemis** [Fallout-London/FOLON-FO4Downgrader#9]
Trying to run the downgrader under wayland leads to the following error:
```
curl: try 'curl --help' for more information
xdg_toplevel#35: error 2: minimum width can't be bigger than the maximum width
The Wayland connection experienced a fatal error: Protocol error
```

Work around is to force X11/XWayland by setting `QT_QPA_PLATFORM=xcb`.

**Comment by kubikill** [comment]
The workaround Athemis posted did not work for me. However, running the downgrader using this command does work for me:
`env WAYLAND_DISPLAY= ./FOLON-Downgrader`

**Comment by GnomeBeans** [comment]
My situation required this on arch

create python venv
python -m venv path/to/venv

source venv/bin/activate

install pip PyQt5

install tk system package from arch repo

To run:
env WAYLAND_DISPLAY= python3 ./FOLON-Downgrader.py

**Comment by coffandro** [comment]
Could one of you try with the following command?
```
git clone https://github.com/Fallout-London/FOLON-FO4Downgrader.git && cd FOLON-FO4Downgrader && chmod +x && ./Run.sh
```

**Comment by coffandro** [comment]
Fixed in with https://github.com/Fallout-London/FOLON-FO4Downgrader/commit/bb85a66a43473554eebf9c9b715d1e290918ba40

**Comment by stoniwankenobii** [comment]
> Fixed in with [bb85a66](https://github.com/Fallout-London/FOLON-FO4Downgrader/commit/bb85a66a43473554eebf9c9b715d1e290918ba40)

Just gave it a try and got this twice when at the download section.

![FOLON Downgrader Error](https://github.com/user-attachments/assets/fdfd1a1c-95f7-46ad-9c71-fd4c6503c19f)

**Comment by coffandro** [comment]
That is a known error on the main branch, will be fixed when BackendSwitch get's merged
